diff --git a/node_modules/@react-aria/utils/dist/main.js b/node_modules/@react-aria/utils/dist/main.js
index dc1c8a0..13760d1 100644
--- a/node_modules/@react-aria/utils/dist/main.js
+++ b/node_modules/@react-aria/utils/dist/main.js
@@ -56,6 +56,10 @@ const $78605a5d7424e31b$export$e5c5a5f917a5871c = typeof window !== 'undefined'
 
 let $8c61827343eed941$var$idsUpdaterMap = new Map();
 function $8c61827343eed941$export$f680877a34711e37(defaultId) {
+	  if (typeof $1Yh1N$react.useId === "function") {
+			let gen = $1Yh1N$react.useId();
+			return defaultId != null ? defaultId : gen;
+		}
     let [value, setValue] = $1Yh1N$react.useState(defaultId);
     let nextId = $1Yh1N$react.useRef(null);
     let res = $1Yh1N$reactariassr.useSSRSafeId(value);
@@ -97,6 +101,9 @@ function $8c61827343eed941$export$cd8c9cb68f842629(idA, idB) {
     return idB;
 }
 function $8c61827343eed941$export$b4cc09c592e8fdb8(depArray = []) {
+	  if (typeof $1Yh1N$react.useId === "function") {
+			return $1Yh1N$react.useId();
+		}
     let id = $8c61827343eed941$export$f680877a34711e37();
     let [resolvedId, setResolvedId] = $19a2307bfabafaf1$export$14d238f342723f25(id);
     let updateId = $1Yh1N$react.useCallback(()=>{
diff --git a/node_modules/@react-aria/utils/dist/module.js b/node_modules/@react-aria/utils/dist/module.js
index 4877c2c..218b2d4 100644
--- a/node_modules/@react-aria/utils/dist/module.js
+++ b/node_modules/@react-aria/utils/dist/module.js
@@ -13,6 +13,10 @@ const $f0a04ccd8dbdd83b$export$e5c5a5f917a5871c = typeof window !== 'undefined'
 
 let $bdb11010cef70236$var$idsUpdaterMap = new Map();
 function $bdb11010cef70236$export$f680877a34711e37(defaultId) {
+	  if (typeof $12uGp$react.useId === "function") {
+			let gen = $12uGp$react.useId();
+			return defaultId != null ? defaultId : gen;
+		}
     let [value, setValue] = $12uGp$useState(defaultId);
     let nextId = $12uGp$useRef(null);
     let res = $12uGp$useSSRSafeId(value);
@@ -54,6 +58,9 @@ function $bdb11010cef70236$export$cd8c9cb68f842629(idA, idB) {
     return idB;
 }
 function $bdb11010cef70236$export$b4cc09c592e8fdb8(depArray = []) {
+	  if (typeof $12uGp$react.useId === "function") {
+			return $12uGp$react.useId();
+		}
     let id = $bdb11010cef70236$export$f680877a34711e37();
     let [resolvedId, setResolvedId] = $1dbecbe27a04f9af$export$14d238f342723f25(id);
     let updateId = $12uGp$useCallback(()=>{
